app/javascript/components/signups/SignUpForm.tsx

```diff
@@ -341,7 +340,29 @@ const SignUp = () => {
                 </select>
               </Form.Item>
 
-              {/* <Divider>Application</Divider> */}
+              <Divider>Availability</Divider>
+              <Row>
+                <Col span={12}>
+                  <Form.Item
+                    name="available_from"
+                    label="From"
+                    rules={[
```

Comment:
For my own edification, does this prop also support regex for formatting validation, too?  In case we want to make sure the input is set up like `01/01/2022`?

Reply:
Looks like it!

There's a section on this page called "pattern regular validation" that has an example:

https://programmer.group/understanding-and-use-of-antdesign-form.html